template/apps/web/src/pages/_app/PageConfig/OnboardingForm/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, memo, useCallback, useMemo, useState } from 'react';
+import { useRouter } from 'next/router';
+import { Group, Stack, Stepper } from '@mantine/core';
+
+import { ShipLightImage } from 'public/images';
+
+import { accountApi } from 'resources/account';
+
+import { GOALS, ROLES } from 'resources/account/account.constants';
```

Comment:
you don't need a separate file `account.constants` in the resources folder. Because it's a single place, where `GOALS` and `ROLES` will be used. Just place it inside OnboardingForm folder